galaxykit/client.py

```diff
@@ -147,6 +127,38 @@ def __init__(
                     container_registry,
                     tls_verify=container_tls_verify,
                 )
+    
+    def _refresh_jwt_token(self):
+        if not self.original_token:
+            self.original_token = self.token
+        else:
+            print("!!!! REFRESHING JWT TOKEN !!!!")
```

Comment:
on second thought, should we maybe use a logger for this? If it appears in stdout, then it could muddy up some of our commands that output useful data, like `galaxykit get users`, etc etc

Reply:
thanks for catching, fixing now